custom/icds_reports/tests/reports/test_issnip_monthly_register.py

```diff
@@ -93,11 +105,13 @@ def test_child_health_monthly_ucr(self):
 
     def test_agg_child_health_monthly(self):
         config = {
-            'awc_id': 'a3',
+            'awc_id': ['a3'],
```

Comment:
Can you add a test that checks multiple AWCs?

Reply:
@emord I added tests, Sorry for the delay.